Source/WebKitLegacy/mac/WebView/WebFeature.h

```diff
@@ -64,7 +64,8 @@ typedef NS_ENUM(NSUInteger, WebFeatureCategory) {
     WebFeatureCategoryNetworking = 6,
     WebFeatureCategoryPrivacy = 7,
     WebFeatureCategorySecurity = 8,
-    WebFeatureCategoryHTML = 9
+    WebFeatureCategoryHTML = 9,
+    WebFeatureCategoryExtensions = 10
```

Comment:
`WebFeatureCategoryExtensions = 10,` to prevent someone else having to change this line when another category is added

Reply:
Alrighty, this should be updated with the latest changes -- thanks!